index.html

```diff
@@ -26,46 +26,246 @@
+					</li>
+				</ul>
+				<button class="primary-btn" type="button">See Project</button>
+			</div>
+			<div class="card">
+				<img
+					class="card-img"
+					src="./images/multipost.jpg"
+					alt="work snapshot"
+				/>
+				<h3 class="card-title">
+					Multi-Post <br />
+					Stories
+				</h3>
+				<div class="card-details">
+					<span class="active-span">CANOPY</span>
+					<img src="./images/circle.svg" alt="grey circle" />
+					<span>Back End Dev</span>
+					<img src="./images/circle.svg" alt="grey circle" />
+					<span>2015</span>
+				</div>
+				<p>
+					A daily selection of privately personalized reads; no accounts or
+					sign-ups required.
+				</p>
+				<ul>
+					<li>
+						<span>html</span>
+					</li>
+					<li>
+						<span>css</span>
+					</li>
+					<li>
+						<span>javaScript</span>
+					</li>
+				</ul>
+				<button class="primary-btn" type="button">See Project</button>
+			</div>
+			<div class="card">
+				<img class="card-img" src="./images/tonic2.jpg" alt="work snapshot" />
+				<h3 class="card-title">Tonic</h3>
+				<div class="card-details">
+					<span class="active-span">CANOPY</span>
+					<img src="./images/circle.svg" alt="grey circle" />
+					<span>Back End Dev</span>
+					<img src="./images/circle.svg" alt="grey circle" />
+					<span>2015</span>
+				</div>
+				<p>
+					A daily selection of privately personalized reads; no accounts or
+					sign-ups required.
+				</p>
+				<ul>
+					<li>
+						<span>html</span>
+					</li>
+					<li>
+						<span>css</span>
+					</li>
+					<li>
+						<span>javaScript</span>
+					</li>
+				</ul>
+				<button class="primary-btn" type="button">See Project</button>
+			</div>
+			<div class="card">
+				<img
+					class="card-img"
+					src="./images/multipost2.jpg"
+					alt="work snapshot"
+				/>
+				<h3 class="card-title">
+					Multi-Post <br />
+					Stories
+				</h3>
+				<div class="card-details">
+					<span class="active-span">CANOPY</span>
+					<img src="./images/circle.svg" alt="grey circle" />
+					<span>Back End Dev</span>
+					<img src="./images/circle.svg" alt="grey circle" />
+					<span>2015</span>
+				</div>
+				<p>
+					A daily selection of privately personalized reads; no accounts or
+					sign-ups required.
+				</p>
+				<ul>
+					<li>
+						<span>html</span>
+					</li>
+					<li>
+						<span>css</span>
+					</li>
+					<li>
+						<span>javaScript</span>
+					</li>
+				</ul>
+				<button class="primary-btn" type="button">See Project</button>
+			</div>
+		</section>
+		<section class="about-me">
+			<div class="about-container">
+				<h2>
+					About <br />
+					Myself
+				</h2>
+				<p class="intro">
+					Hello I’m a software developer! I can help you build a product ,
+					feature or website Look through some of my work and experience! If you
+					like what you see and have a project you need coded, don’t hestiate to
+					contact me.
+				</p>
+				<p class="primary connect">LET’S CONNECT</p>
```

Comment:
- [ ] Kindly make sure to use the `hx` (heading) tag while this `LET'S CONNECT` is a title, it is better if we use a proper tag for presenting the title which makes our works look more professional and increase our page accessibility. :wink: